hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketAllocator.java

```diff
@@ -545,7 +545,7 @@ public void setTo(long free, long used, long itemSize) {
   void logStatistics() {
     IndexStatistics total = new IndexStatistics();
     IndexStatistics[] stats = getIndexStatistics(total);
-    LOG.info("Bucket allocator statistics follow:\n");
+    LOG.info("Bucket allocator statistics follow:");
```

Comment:
This preamble is weird ... "... follow". I like how you put it all on one line at least.

Reply:
I think we should remove this preamble and make each of the below logs just start with something like `"Bucket allocator stats: "`.

unless we add a log message after the for loop that basically says "we finished logging the bucket allocator statistics" it's not like we can use this preamble log line to better find sets of these logs in the noise.

Reply:
I like the @busbey  suggestion